Add support for organization level metrics (#12375)
DataDog/integrations-core: snowflake/README.md

3. [Restart the Agent][6].

#### Collecting Organization Data

> By default, this integration monitors the `ACCOUNT_USAGE` schema, but can also monitor the `ORGANIZATION_USAGE` schema to monitor organization-level metrics.

Reviewer: The usage of _also_ can be misleading. If you set it to collect organization metrics it wont collect account usage metrics, right? Then I think we should say _instead_ instead of _also_. Then mention that you can configure two instances if you want to collect both.

> By default, this integration monitors the `ACCOUNT_USAGE` schema, but can also monitor the `ORGANIZATION_USAGE` schema to monitor organization-level metrics.

Reviewer: The usage of _also_ can be misleading. If you set it to collect organization metrics it wont collect account usage metrics, right? Then I think we should say _instead_ instead of _also_. Then mention that you can configure two instances if you want to collect both.

```suggestion
By default, this integration monitors the `ACCOUNT_USAGE` schema, but it can be set to monitor organization-level metrics instead.
```

Author: Yes, that is correct. I can updated the working to reflect this.

> To collect organization metrics, you can create configure the integration by changing the schema field to `ORGANIZATION_USAGE` and increasing the `min_collection_interval` to reduce the number of queries to Snowflake, as most organization queries have a latency of up to 24 hours.

Reviewer: ```suggestion
To collect organization metrics, you can configure the integration by changing the schema field to `ORGANIZATION_USAGE`. You should also increase the `min_collection_interval` to reduce the number of queries to Snowflake as most organization queries have a latency of up to 24 hours.
```

Note: To monitor organization metrics, your `user` must have the `ORGADMIN` role.

    ```yaml
       - schema: ORGANIZATION_USAGE
         min_collection_interval: 43200
    ```

#### Collecting data for multiple environments

> If you want to collect data for multiple Snowflake environments, add each environment as an instance in your `snowflake.d/conf.yaml` file. For example, if you needed to collect data for two users named `DATADOG_SYSADMIN` and `DATADOG_USER`:

Reviewer: and add an example at the bottom of this section to collect both account and organization metrics

Author: Sure, I can do that!

3. [Restart the Agent][6].

#### Collecting Organization Data

By default, this integration monitors the `ACCOUNT_USAGE` schema, but it can be set to monitor organization-level metrics instead.

To collect organization metrics, you can create configure the integration by changing the schema field to `ORGANIZATION_USAGE` and increasing the `min_collection_interval` to 43200 to reduce the number of queries to Snowflake, as most organization queries have a latency of up to 24 hours.

Note: To monitor organization metrics, your `user` must have the `ORGADMIN` role.

    ```yaml
       - schema: ORGANIZATION_USAGE
         min_collection_interval: 43200
    ```

Additionally, you can monitor both account and organization metrics at the same time:

    ```yaml
    instances:
      - account: example-inc
        username: DATADOG_ORG_ADMIN
        password: '<PASSWORD>'
        role: SYSADMIN
        database: ORGANIZATION_USAGE
        min_collection_interval: 3600

      - account: example-inc
        username: DATADOG_ACCOUNT_ADMIN
        password: '<PASSWORD>'
        role: DATADOG_ADMIN
        database: ACCOUNT_USAGE
        min_collection_interval: 43200

>     ```

Author: Ah yes that's right, I was copying the other example 🤦‍♀️
